### ISSUE TITLE
Tags regex breaks when one of the attributes contains a whitespace

### ISSUE THREAD

**Issue by spuxx1701** [nodecraft/ya-bbcode#162]
Hi there!

It appears that the `tags` regex fails to recognize a tag when that tag has whitespaces within its attributes.

Example: 
```
[quote=123,456,\"Some User\"][b] ... [/b][/quote]
```
Results in a `Array(3) [ "[b]", "[/b]", "[/quote]" ]` tags array. Replacing the whitespace with e.g. an `_` fixes the behavior. I have a usecase where I'm trying to parse BBCodes quotes where the author names can contain whitespaces. I'm not the best with regexes, but maybe you would know how to fix this @Cherry? :)

**Comment by spuxx1701** [comment]
It appears that removing the whitespace matcher `\s` rom the `tags` regex seems to fix the issue for me without introducing any side effects. What's the purpose of that matcher? The `tags` regex would then look like this: `/(\[[^\]^]+])/g`

**Comment by spuxx1701** [comment]
I've provided a PR for this. 163

**Comment by Cherry** [comment]
Thanks for the issue! What HTML would you expect that to produce?

```
<blockquote author="123,456,"Some User""><strong> ... </strong></blockquote>
```

Something like that doesn't feel like valid HTML to me. Should the quotes inside the attribute be escaped? Is that valid? What do you think? Can you add a test in the `test.js` file for this behaviour with your PR?

**Comment by spuxx1701** [comment]
In my case I needed to write a custom parser, but since ya-bbcode didn't recognize the tag in the first place, I needed to solve that first. As you can see [here](https://github.com/spuxx1701/potber/blob/develop/app/services/content-parser.ts#L103), I needed to parse quotes into some more complex HTML consisting of several elements (including an <a> tag and the actual blockquote).

**Comment by spuxx1701** [comment]
I think the default `quote` parser is fine as is, only the `\s` matcher in the regex prevented me from doing what I wanted to do with a custom parser.

**Comment by Cherry** [comment]
Fixed in #163. New version will be released later today 🎉 Thank you!
